### PR TITLE
WARG Onboarding Submission - Jomills Jose Anand

### PR DESCRIPTION
**Pull request by jomillsjoseanand**
Built a classification model using TensorFlow to classify the CIFAR-10 dataset. Generated a plot to check for loss and accuracy as the model trained and achieved an overall accuracy of 78.21%.

This is my onboarding submission for the WARG Computer Vision Team. Check it out and let me know in case there's anything I can change or do better. Thanks! 

image

plot

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,17 +1,198 @@
...
+    xTest = xTest/255.0
+
+    # One hot encoding target values
+    yTrain = tf.keras.utils.to_categorical(yTrain)
+    yTest = tf.keras.utils.to_categorical(yTest)
+
+    return xTrain, xTest, yTrain, yTest
+
+
+def create_model(xTrain, yTest):
```
Comment (rayjinghaolei) [inline main.py:38]: just a small thing here, you can only pass in the required shape for xTrain and yTest instead of passing all of them

Comment (rayjinghaolei) [reply main.py:38]: other wise all good here, good job, this is beautiful code

**Review by rayjinghaolei — CHANGES_REQUESTED** [review]

**Comment by jomillsjoseanand** [comment]
I have updated the function to only accept the shapes of xTrain and yTest as arguments. Kindly check it out and provide me any further feedback. Thanks!

**Comment by rayjinghaolei** [comment]
Good job, welcome to the team, First CV meeting is May 26th @7pm see you there
